phpBB/includes/functions_messenger.php

```diff
@@ -44,10 +42,6 @@ function messenger($use_queue = true)
 
 		$this->use_queue = (!$config['email_package_size']) ? false : $use_queue;
 		$this->subject = '';
-
-		// Determine EOL character (\n for UNIX, \r\n for Windows and \r for Mac)
```

Comment:
`PHP_EOL` was introduced in PHP 5.0.2. So there is no need to have a custom version.